src/core/css/inc/mixins.scss

```diff
@@ -164,41 +163,71 @@ $default-arrow-margin: 3px;
 }
 
 /* Button */
-@mixin button($background: $button-background-color,
-$color: $text-color-message) {
+@mixin button(
+  $background,
+  $background-active: #fff,
+  $background-hover: #fff,
+  $border-color: transparent,
+  $border-radius: $border-radius-s,
+  $color: $white
+) {
   @include font-regular();
 
   background: $background;
-  border-radius: $border-radius-default;
-  border: 0;
-  color: $color;
-  cursor: pointer;
+  border-radius: $border-radius;
+  border: 1px solid $border-color;
   display: inline-block;
-  font-size: $font-size-m;
-  line-height: 1.5em;
   margin: 0;
-  padding: 0.5em 1em;
+  min-height: 39px;
+  padding: 10px 14px 8px;
   text-align: center;
   text-decoration: none;
+  text-shadow: 1px 1px 0 transparentize(invert($color), 0.5);
   transition-duration: $transition-short;
-  transition-property: background-color;
+  transition-property: background-color border-color;
+  vertical-align: middle;
 
-  &:active,
-  &:hover,
-  &:focus {
-    @if lightness($background) < 5% {
-      background: lighten($background, 9%);
-    } @else {
-      background: darken($background, 3%);
+  .Icon {
+    @include margin-end(6px);
+
+    height: 16px;
+    vertical-align: top;
+    width: 16px;
+  }
+
+  &.Button--small {
+    &,
```

Comment:
Ah ok, maybe something to look at if cascade can be used in place of the mixin later.